packages/blade/src/components/ActionList/ActionListHeader.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import styled from 'styled-components';
+import { componentIds } from './componentIds';
+import Box from '~components/Box';
+import type { IconComponent } from '~components/Icons';
+import { makeSize } from '~utils';
+import { BaseText } from '~components/Typography/BaseText';
+
+const StyledActionListHeader = styled(Box)((props) => {
+  return {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    padding: `${makeSize(props.theme.spacing[3])} ${makeSize(props.theme.spacing[5])}`,
+    backgroundColor: props.theme.colors.brand.gray.a50.lowContrast,
+    margin: `-${makeSize(props.theme.spacing[3])} -${makeSize(props.theme.spacing[3])} ${makeSize(
+      props.theme.spacing[3],
+    )}`,
+  };
+});
+
+const ActionListHeader = (props: { title: string; leading?: React.ReactNode }): JSX.Element => {
+  return (
+    <StyledActionListHeader>
+      <Box>{props.leading}</Box>
+      <Box paddingLeft="spacing.3" paddingRight="spacing.3">
+        <BaseText color="surface.text.subdued.lowContrast" fontStyle="italic" fontSize={50}>
```

Comment:
Possible to use the Text component instead of BaseText?

Reply:
Ohhh rightt! I forgot about caption variant. I was trying to find italic text there 🙈 changed it in upcoming commit